graphql_auth/settings.py

```diff
@@ -53,7 +53,16 @@
     "EMAIL_TEMPLATE_SECONDARY_EMAIL_ACTIVATION": "email/activation_email.html",
     "EMAIL_TEMPLATE_PASSWORD_RESET": "email/password_reset_email.html",
     # query stuff
-    "USER_NODE_EXCLUDE_FIELDS": ["password", "is_superuser"],
+    "USER_NODE_FIELDS": [
+        "email",
+        "username",
+        "last_login",
+        "isStaff",
```

Comment:
Are we looking for Django field names or Node field names here? The list is inconsistent.

Reply:
These are the fields we want to be available in the user node.

Reply:
I mean, some of the fields has underscore and others are camel case.

Reply:
You are right, I didn't see that. Should be with underscore.